### ISSUE TITLE
Save Default Param Values

### ISSUE THREAD

**Issue by kaalleen** [inkstitch/inkstitch#1069]
Discussed here: #1032

**Comment by lexelby** [comment]
Great idea!

**Comment by danielkschneider** [comment]
Yes great idea. But preferably with an option to either append to the old file or erase it.
It would  be of great help to beginners to be able to distribute a set of standard settings.
IMHO, one also could include a few in the distribution, for example:
- neutral tissue - fill
- neutral tissue - satin
- neutral tissue - satin topstitch
-  .....
- wovens - fill
- wovens - satin
- ....
- knits - fill
- knits - satin
